### PR TITLE
Storage tests

### PR DESCRIPTION
**Pull request by louislai**

### CODE REVIEW

File: src/main/java/seedu/toluist/storage/JsonStorage.java [hunk 1]
```diff
@@ -107,6 +97,16 @@ private boolean saveNotAffectingHistory(TodoList todoList, String storagePath) {
         return new Pair<>(todoList, times - steps);
     }
 
+    private boolean saveNotAffectingHistory(TodoList todoList, String storagePath) {
```
Comment (Melvin-Tan) [inline src/main/java/seedu/toluist/storage/JsonStorage.java:100]: By moving this down, got any difference meh? o.o

File: src/test/java/seedu/toluist/model/TodoListTest.java [hunk 2]
```diff
@@ -64,6 +64,26 @@ public void constructor_withStorage_loadTasks() {
...
+    public void constructor_replaceCurrentTodoList() {
+        TodoList anotherTodoList = new TodoList();
+        assertEquals(anotherTodoList, TodoList.load());
+    }
+
+    @Test
+    public void save_replaceCurrentTodoList() {
+        TodoList anotherTodoList = new TodoList();
+        anotherTodoList.add(new Task("ask"));
+        TodoList emptyTodoList = new TodoList();
```
Comment (Melvin-Tan) [inline src/test/java/seedu/toluist/model/TodoListTest.java:76]: this is not used and codacy complains

Comment (louislai) [reply src/test/java/seedu/toluist/model/TodoListTest.java:76]: its actually `used`. Look at the previous test case, constructitng an object is doing sth

File: src/main/java/seedu/toluist/storage/JsonStorage.java (hunk 1 above)
Comment (louislai) [reply src/main/java/seedu/toluist/storage/JsonStorage.java:100]: so private methods are grouped together

**Review by Melvin-Tan — APPROVED** [review]
Ok then, merge! :D

**Pull request merged**